portfolio/src/main/webapp/index.html

```diff
@@ -7,11 +7,24 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+    <div class="content">
+      <h1> Alisha Kewalramani</h1>
+      <nav>
+          <ul id = "horizontal-list">
```

Comment:
nit: use an `id` that is descriptive of its purpose, e.g. `"navigation"` or `"links"`

(generally HTML tags/attributes should describe the semantic document structure, which is separate from its visual presentation)